resources/cdk/cognito_scenario_user_pool_with_mfa/README.md

```diff
@@ -0,0 +1,118 @@
+# Cognito user pool with MFA scenario resources
+
+## Overview
+
+Creates AWS resources for Amazon Cognito scenarios. The scripts in this
+example create the following resources:
+ 
+* An Amazon Cognito user pool that is configured to:
+    * allow self sign-up
+    * verify by email
+    * require multi-factor authentication (MFA)
+    * allow device tracking
+* An associated client application that can authenticate with the user pool. 
+
+## ⚠️ Important
+
+* Running this code might result in charges to your AWS account. 
+* This code is not tested in every AWS Region. For more information, see [AWS Regional Services](https://aws.amazon.com/about-aws/global-infrastructure/regional-product-services).
+
+## Deploying resources
+
+You can use the AWS Cloud Development Kit (AWS CDK) or the AWS Command Line Interface
+(AWS CLI) to deploy and destroy the resources for this example.
+
+### Deploying with the AWS CDK
+
+To deploy with the AWS CDK, you must install [Node.js](https://nodejs.org) and the 
+[AWS CDK](https://docs.aws.amazon.com/cdk/v2/guide/getting_started.html).
+
+This example was built and tested with AWS CDK 2.29.1.
+
+Deploy AWS resources by running the following at a command prompt in the
+`resources/cdk/cognito_scenario_user_pool_with_mfa` folder:
+
+```
+npm install
+cdk deploy
+```
+
+The stack takes a few minutes to deploy. When it completes, it prints output like 
+the following:
+
+```
+Outputs:
+doc-example-cognito-scenario-signup-user-with-mfa.ClientId = 123456789eo6cet40krEXAMPLE
+doc-example-cognito-scenario-signup-user-with-mfa.UserPoolId = us-west-2_CzEXAMPLE
+```
+
+### Deploying with the AWS CLI 
+
+To deploy with the AWS CLI you must first install the 
```

Comment:
To deploy with the AWS CLI, you must first install the

Reply:
fixed